docs/topics/secured-booster-minishift.adoc

```diff
@@ -1,7 +1,8 @@
 = Configuring Your {OpenShiftLocal}
-Before you can run the Secured Booster, you must update your {OpenShiftLocal} from the default configuration. You only need to do this once for the secured booster missions.
+Before you can run the Secured Booster, you must update your {OpenShiftLocal} from the default configuration. 
+You only need to do this once for any of the secured mission boosters.
+
```

Comment:
I guess this empty line will make a paragraph break? That's probably not what we want. Otherwise LGTM.